open-metadata-implementation/access-services/discovery-engine/discovery-engine-server/src/main/java/org/odpi/openmetadata/accessservices/discoveryengine/handlers/DiscoveryConfigurationHandler.java

```diff
@@ -283,11 +283,11 @@ public  DiscoveryEngineProperties getDiscoveryEngineByName(String    userId,
         invalidParameterHandler.validateUserId(userId, methodName);
         invalidParameterHandler.validatePaging(startingFrom, maximumResults, methodName);
 
-        List<EntityDetail> retrievedEntities = repositoryHandler.getEntitiesByType(userId,
-                                                                                   DiscoveryEnginePropertiesMapper.DISCOVERY_ENGINE_TYPE_GUID,
-                                                                                   startingFrom,
-                                                                                   maximumResults,
-                                                                                   methodName);
+        List<EntityDetail> retrievedEntities = repositoryHandler.getEntityByType(userId,
+                                                                                 DiscoveryEnginePropertiesMapper.DISCOVERY_ENGINE_TYPE_GUID,
```

Comment:
This seems to be restoring the old name of the method

Reply:
I think this comment refers to an outdated file. This file is not touched in the latest code change for this PR.